src/registered.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs-extra');
+const low = require('lowdb');
+const FileSync = require('lowdb/adapters/FileSync');
+const amministrazioni = require('../public/assets/data/authorities.db.json');
+
+const whitelistFile = 'private/data/whitelist.db.json';
+fs.ensureFileSync(whitelistFile);
```

Comment:
Non serve anche qui un qualche tipo di atomicità sulla scrittura? Non siamo preoccupati da richieste concorrenti?

Reply:
https://github.com/italia/onboarding.developers.italia.it/pull/10/commits/b99a7d11f9d731a07f85585cb5609252c590cf9c